DataTable.svelte

```diff
@@ -75,6 +79,12 @@
 
   $: templateColumns = `max-content ${columns.map(getColumnWidth).join(' max-content ')} max-content`
 
+  const options: string[] = ['10', '30', '50', '100']
```

Comment:
nits: It's hard to know what the options are, so let's be explicit.
e.g. `options` -> `pageSizeOptions`